styles/_wysiwyg.scss

```diff
@@ -0,0 +1,185 @@
+/* WYSIWYG styling inside certain elements only */
+
+.article-body {
+  /* LINKS */
+
+  /* stylelint-disable-next-line selector-max-specificity */
+  a[href*='//']:not([href*='cloudinary.com']):not([href*='getjobber.com']) {
+    /* stylelint-disable-next-line selector-max-specificity */
+    &::after {
+      content: url("data:image/svg+xml; utf8, <svg xmlns='http://www.w3.org/2000/svg' width='15' height='15' viewBox='0 0 15 15' fill='none' stroke='black' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'><path d='M18 14v5a2 2 0 01-2 2H5a2 2 0 01-2-2V8c0-1.1.9-2 2-2h5m5-3h6v6m-11 5L20.2 3.8' fill-rule='evenodd'/></svg>");
+      max-width: 15px;
+      height: auto;
+      margin-left: 5px;
+      /* stylelint-disable-next-line max-nesting-depth */
+      @media print {
+        content: ' [' attr(href) ']';
+        color: $black;
+      }
+    }
+  }
+
+  a {
+    @media print {
+      border-bottom: dotted 1px $grey--light;
+      box-shadow: none;
+      color: $grey--light;
+      /* stylelint-disable-next-line max-nesting-depth */
+      &::after {
+        content: ' [' attr(href) ']';
+        color: $black;
+      }
+    }
+  }
+
+  .popup {
+    box-shadow: none;
+
+    &:hover,
+    &:active,
+    &:focus {
+      box-shadow: none;
+    }
+    @media print {
+      cursor: not-allowed;
+    }
+  }
+
+  /* MEDIA */
+  img {
+    max-width: 100%;
+    height: auto;
+    border: solid $border--width $grey;
+
+    &.mfp-img {
+      border: none;
+    }
```

Comment:
new selectors should come after media query so that the compiled order makes sense

Reply:
Oops! Accidentally merged. It's hard to tell who has approved! 
I fixed this in #11